cli/js/web/url.ts

```diff
@@ -503,20 +510,72 @@ function charInUserinfoSet(c: string): boolean {
   return charInPathSet(c) || ["\u0027", "\u002F", "\u003A", "\u003B", "\u003D", "\u0040", "\u005B", "\u005C", "\u005D", "\u005E", "\u007C"].includes(c);
 }
 
+function charIsForbiddenInHost(c: string): boolean {
+  // prettier-ignore
```

Comment:
seems unnecessary to not format

Reply:
Each of these functions just check inclusion in sets of non-human-readable char codes copied from the spec. I'd rather keep them all to one line.

Reply:
If you insist